functions/Get-DbaTraceFlag.ps1

```diff
@@ -0,0 +1,96 @@
+﻿FUNCTION Get-DbaTraceFlag {
+	<#
+		.SYNOPSIS
+			Gets SQL Trace Flags information for each instance(s) of SQL Server.
```

Comment:
Would just say `Get global Trace Flag(s) information for each instance(s) of SQL Server.`